CMakeLists.txt

```diff
@@ -1,15 +1,18 @@
-cmake_minimum_required (VERSION 2.6)
+cmake_minimum_required (VERSION 3.1)
```

Comment:
Is this the real minimum-required? I don't immediately see any new features used.

Reply:
This is required for the set(CMAKE_CXX_STANDARD 11).